packages/jest-matchers/package.json

```diff
@@ -12,5 +12,10 @@
     "jest-matcher-utils": "^19.0.0",
     "jest-message-util": "^19.0.0",
     "jest-regex-util": "^19.0.0"
+  },
+  "devDependencies": {
+    "ansi-regex": "^2.1.1",
```

Comment:
Can you make ansi-styles and ansi-regex devDependencies of the top level package.json instead? jest-snapshot should stay here but please use the latest version (19.0.2).